src/routes.js

```diff
@@ -78,19 +89,71 @@ const Routes = () => {
-          <Route exact path={"/manageusers"} component={AllowManageUser(ManageUsers)} />
-          <Route exact path={"/dashboard"} component={UserIsNotAllowedUserDashboard(Dashboard)} />
-          <Route exact path={"/dashboard/my_feed"} component={UserIsAllowedUserDashboard(MyFeed)} />
-          <Route exact path={"/profile"} component={UserIsAllowedUserDashboard(Profile)} />
-          <Route exact path={"/organization"} component={UserIsAllowOrgManager(Organization)} />
-          <Route exact path={"/tutorials"} component={UserIsAllowedUserDashboard(MyTutorials)} />
-          <Route exact path={"/tutorials/:owner/:tutorial_id"} component={UserIsAllowedUserDashboard(ViewTutorial)} />
-          <Route exact path={"/user/:handle"} component={UserIsAllowedUserDashboard(ProfileView)} />
-          <Route exact path={"/org/:handle"} component={UserIsAllowedUserDashboard(ViewOrganization)} />
-          <Route exact path={"/editor"} component={UserIsAllowedUserDashboard(Editor)} />
+          <Route
+            exact
+            path={"/login"}
+            render={(props) => <AuthPage {...props} type={"login"} />}
+          />
+          <Route
+            exact
+            path={"/signup"}
+            render={(props) => <AuthPage {...props} type={"signup"} />}
+          />
+          <Route
+            exact
+            path={"/forgotpassword"}
+            render={(props) => <AuthPage {...props} type={"forgotpassword"} />}
+          />
+          <Route
+            exact
+            path={"/manageusers"}
+            component={AllowManageUser(ManageUsers)}
+          />
+          <Route
+            exact
+            path={"/dashboard"}
+            component={UserIsNotAllowedUserDashboard(Dashboard)}
+          />
+          <Route
+            exact
+            path={"/dashboard/my_feed"}
+            component={UserIsAllowedUserDashboard(MyFeed)}
+          />
+          <Route
+            exact
+            path={"/profile"}
+            component={UserIsAllowedUserDashboard(Profile)}
+          />
+          <Route
+            exact
+            path={"/organization"}
+            component={UserIsAllowOrgManager(Organization)}
+          />
+          <Route
+            exact
+            path={"/tutorials"}
+            component={UserIsAllowedUserDashboard(MyTutorials)}
+          />
+          <Route
+            exact
+            path={"/tutorials/:owner/:tutorial_id"}
+            component={UserIsAllowedUserDashboard(ViewTutorial)}
+          />
+          <Route
+            exact
+            path={"/user/:handle"}
+            component={UserIsAllowedUserDashboard(ProfileView)}
+          />
+          <Route
+            exact
+            path={"/org/:handle"}
+            component={UserIsAllowedUserDashboard(ViewOrganization)}
+          />
+          <Route
+            exact
+            path={"/editor"}
+            component={UserIsAllowedUserDashboard(Editor)}
+          />
```

Comment:
I prefer the previous way that we used to have in here more

Reply:
ok I'll change prettier format